locations/spiders/mcdonalds_bg.py

```diff
@@ -1,40 +1,46 @@
-import re
+import json
 
-import scrapy
+from scrapy import FormRequest, Spider
 
-from locations.items import Feature
+from locations.categories import apply_yes_no
+from locations.dict_parser import DictParser
 from locations.spiders.mcdonalds import McDonaldsSpider
 
 
-class McDonaldsBGSpider(scrapy.Spider):
+class McDonaldsBGSpider(Spider):
     name = "mcdonalds_bg"
     item_attributes = McDonaldsSpider.item_attributes
     allowed_domains = ["mcdonalds.bg"]
-    start_urls = ("http://mcdonalds.bg/map/",)
+    start_urls = ["http://mcdonalds.bg/restaurants/"]
 
     def parse(self, response):
-        stores = response.css(".restaurant-info")
-        for store in stores:
-            ref = store.xpath(".//@data-post_id").extract_first()
-            city = store.xpath(".//@data-city_title").extract_first()
-            lat = store.xpath(".//@data-latitude").extract_first()
-            lon = store.xpath(".//@data-longitude").extract_first()
-            address = store.xpath(".//p[1]/text()").extract_first()
-
-            phone = store.xpath(".//p[2]/text()").extract_first()
-            if phone:
-                match = re.search(r"Тел.(.*)", phone)
-                if match:
-                    phone = match.groups()[0].strip()
-
-            properties = {
-                "ref": ref,
-                "phone": phone if phone else "",
-                "lon": lon,
-                "lat": lat,
-                "city": city,
-                "name": "McDonalds",
-                "addr_full": address,
-            }
-
-            yield Feature(**properties)
+        name_ids = response.xpath("//strong[@class='restaurant-item__title']/@data-restaurant-id").extract()
+
+        for id in name_ids:
+            yield FormRequest(
```

Comment:
It may not matter in this instance, but I think `method="POST"` may be worthwhile to add as an argument, as that is what `https://mcdonalds.bg/restaurants/` is using.

Reply:
It must have defaulted to POST method as that what works in Postman. But just to be sure it uses POST I added. Thank you.